torchaudio/functional/filtering.py

```diff
@@ -948,6 +949,10 @@ def lfilter(
                                 Lower delays coefficients are first, e.g. ``[b0, b1, b2, ...]``.
                                 Must be same size as a_coeffs (pad with 0's as necessary).
         clamp (bool, optional): If ``True``, clamp the output signal to be in the range [-1, 1] (Default: ``True``)
+        batching (bool, optional): Activate when coefficients are in 2D. If ``True``, then waveform should be at least
```

Comment:
```suggestion
        batching (bool, optional): Effective only when coefficients are 2D. If ``True``, then waveform should be at least
```

Reply:
Thanks,. Should I make another MR to fix the docstring?

Reply:
If you have time, please!